src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -181,10 +181,6 @@
       <Context>File;ProjectSubscriptionService</Context>
     </PropertyPageSchema>
 
-    <PropertyPageSchema Include="$(ManagedXamlResourcesDirectory)Resource.BrowseObject.xaml">
```

Comment:
>Resource.BrowseObject.xaml [](start = 65, length = 26)

Out of curiosity, which item is hidden in the tree (mentioned in your PR description)? I'm not sure what this BrowseObject is or how it relates to your PR description.

Reply:
Resource.BrowseObject describes the properties that will show in the Property window of the `<Resource  />` item. Currently, `<Resource/>` and `<Page/>` items don't show in the tree (can be caused by missing "BrowseObject") but its unrelated this exact change. These browse objects were deliberately removed in lieu of other code that was supposed to create them on the fly - this isn't working.